src/main/java/game/RacingGame.java

```diff
@@ -0,0 +1,79 @@
+package game;
+
+import java.util.Random;
+
+public class RacingGame {
+
+    public final Integer POINT_FORWARD = POINT;
+
+    public final static Integer POINT = 4;
+
+    /**
+     * 자동차 경주 게임
+     * @param cars
+     * @param count
+     */
+    public void racing(int cars, int count){
+
+        for(int i = 0 ; i < count ; i++){
+            ResultView.create().racingResultView(move(cars,count));
+        }
+    }
+
+    /**
+     * n개 자동차 경주 결과
+     * @param cars
+     * @param count
+     * @return
+     */
+    public int[] move(int cars, int count) {
+
+        int[] carPositions = new int[cars];
+
+        for (int i = 0; i < cars; i++) {
+            carPositions[i] = getMovePoint(count);
+        }
+
+        return carPositions;
+    }
+
+    /**
+     * 시도횟수에 대한 이동 횟수 구하기
+     *
+     * @return
+     */
+    public int getMovePoint(int count) {
+
+        int result = 0;
+        for(int i =0 ; i < count ;i++){
```

Comment:
```suggestion
        for(int i = 0; i < count; i++){
```
혹, ide 로 intellij 를 사용하신다면 커밋 전 reformat 을 적용(opt + commad + L) 해보는 연습도 같이 해보시면 좋을 것 같습니다!